### ISSUE TITLE
branch 2.11 seems to be broken

### ISSUE THREAD

**Issue by mickael-oclock** [lexik/LexikJWTAuthenticationBundle#841]
First thank you for all the work you do
after a clean installation, version 2.11 returns an error 

```Attempted to load class "JoseEncoder" from namespace "Lcobucci\JWT\Encoding".```

I don't have time to investigate the causes, but after a downgrade to 2.10 everything works again.  

Thanks

**Comment by chalasr** [comment]
Hey, 

Can you please share the full stack trace?

**Comment by chalasr** [comment]
and the version of `lcobucci/jwt` you are using (`composer show lcobucci/jwt`)?

**Comment by chalasr** [comment]
Closing due to the lack of feedback.

**Comment by Barry3D** [comment]
Leaving this here if someone else has this error in the future.

In my case I got the same error and it wasn't fixed by downgrading.

By trying to update lcobucci/jwt I got an error that I was missing `php-mbstring` and thus could not install it correctly. (There was no warning or error while installing lexik :cry:)

In the end I just installed `php-mbstring` and retried `composer update lcobucci/jwt`.
